README.md

```diff
@@ -126,39 +132,47 @@ modifications necessary.
 
 Here is an example of a very simple bootstrap definition file for CentOS:
 
+```yaml
```

Comment:
The highlighting works here, but the file is definitely not yaml. I'm a little concerned that someone might think the format is yaml if they see this.